### PR TITLE
community: update readme spanish translation (fix #5851)

### LINKED ISSUE #5985: The spanish readme file is outdated
Hi! when I was reviewing the issue #5851 I saw the Spanish version of the Readme file and realized that it is out of date, particularly in the QuickStart section, I am attaching an image that describes the problem.

Screenshot from 2020-10-11 21-36-42

I would like to work on this issue if there are not exists problems

### PR DESCRIPTION
**Pull request by y-martinez**
The file is outdated, the section quickstart was incomplete, specifically, these were the changes:

- Update the section QuickStart with Hasura Cloud and Render
- Update the links of the others one-click deployment options
- Change the name of the file
- Add shield of release at beginning of the file
- Update the flag to Spain flag

### Changelog

- [X] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components

- [X] Docs
- [X] Community Content

### Related Issues

Fix #5851 
Fix #5985 

#### Catalog upgrade

Does this PR change Hasura Catalog version?
- [X] No

#### Metadata

Does this PR add a new Metadata feature?
- [X] No

#### GraphQL
- [X] No new GraphQL schema is generated

#### Breaking changes

- [X] No Breaking changes

### CODE REVIEW

**Comment by tylerhannan** [comment]
Just an update that we are looking for a native Spanish speaker to help provide review for this...

**Comment by y-martinez** [comment]
> Just an update that we are looking for a native Spanish speaker to help provide review for this...

Hello! Actually I'm from Venezuela 🇻🇪, so Spanish is my native language. Why do you think I am a non Spanish speaker? I am aware I need another native Spanish speaker to make a review of my PR, I thought as a native speaker I can make reviews of another Spanish translation. Please let me know any comments about what is wrong with this.

Regards :)

**Comment by tylerhannan** [comment]
@y-martinez - I am not questioning your language ability. Just making sure you know that we are working to find another reviewer to give the PR a +1 ;)

Given the amount of churn we've had on translations recently, there is benefit in having additional people review. That is all.

Your time and effort on this is super appreciated. I've already asked in the spanish speaking channel on our discord to help locate another reviewer.

File: translations/README.korean.md [hunk 1]
```diff
@@ -42,22 +42,27 @@ Hasura는 Postgres를 지원하는 GraphQL 앱을 만들거나 Postgres를 사
...
-- [지원 및 트러뷸 슈팅](#지원__트러뷸슈팅)
-- [컨트리뷰트](#기여)
-- [브랜드 자산들](#브랜드_자산)
-- [라이선](#라이선스)
-- [번역](#번역)
+  - [지원__트러뷸슈팅](#지원__트러뷸슈팅)
+  - [기여](#기여)
+  - [브랜드_자산](#브랜드_자산)
+  - [라이선스](#라이선스)
+  - [번역](#번역)
```
Comment (codingkarthik) [inline translations/README.korean.md:66]: Maybe the korean changes can be made in a separate PR?

File: translations/README.spanish.md [hunk 2]
```diff
@@ -42,38 +43,43 @@ Leer más en [hasura.io](https://hasura.io) y en [docs](https://hasura.io/docs).
...
-1. Da click en el siguiente botón para desplegar el motor GraphQL en Heroku con el add-on gratuito de Postgres:
+1. Haga click en el siguiente botón para desplegar el motor GraphQL en Hasura Cloud, incluyendo el complemento de Postgres o utilizando una base de datos de Postgres existente:
 
-    [![Deploy to Heroku](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/hasura/graphql-engine-heroku)
+    [![Deploy to Hasura Cloud](https://graphql-engine-cdn.hasura.io/img/deploy_to_hasura.png)](https://cloud.hasura.io/)
 
 2. Abre la consola de Hasura
 
-   Visita `https://<app-name>.herokuapp.com` (*reemplaza \<app-name\> con el nombre de tu aplicacion*) para abrir la consola de administrador.
+    Haga clic en el botón "Iniciar consola" para abrir la consola Hasura.
```
Comment (codingkarthik) [inline translations/README.spanish.md:82]: Maybe it should be "Initialize console" because it will be that way in the hasura console, IIUC.

**Review by codingkarthik — COMMENTED** [review]
I'm not a native spanish speaker, but found some minor nit-picks in the PR.

**Comment by Stefmore02** [comment]
Hey @y-martinez!

Thanks for this work; due to the length it's been open, some of the information is stale. However, we're going to use your contributions to improve the Spanish README, with the most relevant content, prontísimo!
